[Edit] README: Improve "How do I Contribute?" section (#4466)
Codecademy/docs: README.md

> There are many ways to contribute to Docs:

Reviewer: ```suggestion
Here's how you can contribute to Codecademy Docs:
```

1. Get assigned to an Issue of your interest.
   - Ask to be assigned to one of the [available Issues](https://github.com/Codecademy/docs/issues?q=is%3Aissue+is%3Aopen+-label%3Aclaimed).
   - Submit a typo/bug in [GitHub Issues](https://github.com/Codecademy/docs/issues).

> 2. Submit a Pull Request for your assigned Issue.

Reviewer: Can we include a step explicitly mentioning that you should write an entry?

I.e.: 2. write your entry

Reviewer: Use proper grammar of course.
